src/nearly_restful/04_svc2_restful_auto_service/restful_auto_service/data/repository.py

```diff
@@ -24,7 +24,8 @@ class Repository:
 
     @classmethod
     def all_cars(cls, limit=None):
-        cls.__load_data()
+        if not cls.__car_data:
```

Comment:
This line is a duplicate. Can you remove the test and just leave it as it was `cls.__load_data()`?